src/server/services/email.service.js

```diff
@@ -0,0 +1,57 @@
+import nodemailer from 'nodemailer';
+// import google from 'googleapis';
+const { google } = require('googleapis');
+
+export const createUniqueString = () => {
+  const len = 8;
+  let randStr = '';
+  for (let i = 0; i < len; i += 1) {
+    const ch = Math.floor((Math.random() * 10) + 1);
+    randStr += ch;
+  }
+  return randStr;
+};
+
+export const sendEmail = (email, uniqueString) => {
+  const { OAuth2 } = google.auth;
+
+  const myOAuth2Client = new OAuth2(process.env.EMAIL_CLIENT_ID, process.env.EMAIL_CLIENT_SECRET, 'https://developers.google.com/oauthplayground');
+
+  myOAuth2Client.setCredentials({ refresh_token: process.env.EMAIL_CLIENT_REFRESH_TOKEN });
+
+  const myAccessToken = myOAuth2Client.getAccessToken();
+  const transporter = nodemailer.createTransport({
```

Comment:
You don't need to create transport for each email. move the transporter outside this method and use it to send emails.